modules/uhsg_search/uhsg_search.install

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for uhsg_search.
+ *//**
```

Comment:
Something funky going on with this comment line :)

Reply:
Oops.. Fixed. I blame phpcb for joining 2 separate comments.